server/service/src/print/label.rs

```diff
@@ -125,3 +151,31 @@ pub fn host_status(settings: LabelPrinterSettingNode) -> Result<String> {
     let printer = Jetdirect::new(settings.address, settings.port);
     printer.send_string("~HS".to_string(), Mode::Sgd)
 }
+
+// Format a field for the FD command in ZPL
```

Comment:
Tangential but I wonder how hard it'll be to support fonts for Arabic or Russian...

Reply:
Sounds like it's all possible.